src/pages/view-test-result/components/rekey-details-card/rekey-details.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, Input } from '@angular/core';
+import { RekeyDetailsModel } from './rekey-details-card.model';
+
+@Component({
+  selector: 'rekey-details-card',
+  templateUrl: 'rekey-details-card.html',
+})
+export class RekeyDetailsCardComponent {
+
+  @Input()
+  data: RekeyDetailsModel;
```

Comment:
`data` is very generic name. We could use a more verbose one like `rekeyDetails`

Reply:
I agree. However, this follows the convention on all of the other components within the view-test-result page. Happy to change it but are we comfortable changing it for the rekey components only and leaving the others as is?